pkg/procs/info_test.go

```diff
@@ -0,0 +1,115 @@
+package procs_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/openshift-kni/debug-tools/pkg/procs"
+)
+
+var nullLog = log.New(ioutil.Discard, "", 0)
+
+func TestEmpty(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatalf("creating temp dir %v", err)
+	}
+	defer os.RemoveAll(dir) // clean up
+
+	ph := procs.New(nullLog, dir)
+	pidInfos, err := ph.ListAll()
+	if err != nil {
+		t.Errorf("ListAll(%s) failed: %v", dir, err)
+	}
+	if len(pidInfos) > 0 {
+		t.Errorf("found unexpected entries: %v", pidInfos)
+	}
+}
+
+func TestSingleProcSingleThread(t *testing.T) {
+	dir, err := ioutil.TempDir("", "test")
+	if err != nil {
+		t.Fatalf("creating temp dir %v", err)
+	}
+	//	defer os.RemoveAll(dir) // clean up
```

Comment:
Do we need it?

Reply:
Yep, we do. Missed to update when I was deep(er) in dev mode. Good catch, thanks.